src/System.Security.Cryptography.Encoding/tests/Asn1/Writer/WriteCharacterString.cs

```diff
@@ -494,5 +494,57 @@ protected void VerifyWrite_Span_NonEncodable(string input)
                 Assert.Throws<EncoderFallbackException>(() => WriteSpan(writer, input.AsSpan()));
             }
         }
+
+        protected void WriteAfterDispose_Span(bool empty)
+        {
+            using (AsnWriter writer = new AsnWriter(AsnEncodingRules.DER))
+            {
+                if (!empty)
+                {
+                    writer.WriteNull();
+                }
+
+                writer.Dispose();
+
+#if NETCOREAPP
```

Comment:
What's the change that causes these `#if`s to be needed?

Is `NETCOREAPP` ever defined? I think that it's the lowercase version (`netcoreapp`) that comes "for free"

Reply:
There's some build configuration where some implicit Span conversations are not available. I assumed it was some NetFX build, but I will recheck.

Reply:
Right.  But the tests presumably worked before... so my question is why did they work before this change and now fail to compile?  I don't think there's any conditional API in the reader/writer.

Reply:
These are new tests that were not there before.

Reply:
Three of them use the `BigInteger.ToByteArray` overload with `isBigEndian: true`. The other one uses implicit conversion from `string` to `ReadOnlySpan<char>`. Neither of those are in .NET Standard 2.0.

Reply:
I fixed the name of the definition and changed the project to build both as `netstandard` and `netcoreapp`.